pvr.nextpvr/addon.xml.in

```diff
@@ -169,7 +169,7 @@
     <disclaimer lang="zh_CN">此插件仍在开发中，请自行把握试用风险。</disclaimer>
     <disclaimer lang="zh_TW">這個程式尚在開發中。使用上您須自行考慮到可能有的風險。</disclaimer>
     <platform>@PLATFORM@</platform>
-    <license>GPL-2.0</license>
+    <license>GPL-2.0-or-later</license>
```

Comment:
I didn’t add this in the other pvr addon 😔

Reply:
NP, I correct it the next days.